packages/react-native-sdk/src/providers/MediaStreamManagement.tsx

```diff
@@ -71,121 +43,74 @@ export const MediaStreamManagement = ({
   );
 
   const [{ initialAudioEnabled, initialVideoEnabled }, setInitialDeviceState] =
-    useState({
-      initialAudioEnabled: !!propInitialAudioEnabled,
-      initialVideoEnabled: !!propInitialVideoEnabled,
+    useState<MediaDevicesInitialState>({
+      initialAudioEnabled: undefined,
+      initialVideoEnabled: undefined,
     });
```

Comment:
need not change this part though.. the previous one is alright according to me.. its initilaised with the prop that is first passed and never undefined after